static-assets/components/cstudio-dialogs/uploadS3-dialog.js

```diff
@@ -192,33 +192,9 @@ CStudioAuthoring.Dialogs.UploadS3Dialog = CStudioAuthoring.Dialogs.UploadS3Dialo
         me.callback.success(uploaded);
         CStudioAuthoring.Dialogs.UploadS3Dialog.closeDialog();
       },
-      onError: function ({ response }) {
-        const res = JSON.parse(response.body.response).response,
-          errorMsg = `${res.message}. ${res.remedialAction}`;
-
+      onError: function () {
         me.uploadingFile = false;
         $('#uploadCancelButton').attr('disabled', false);
-
-        CStudioAuthoring.Operations.showSimpleDialog(
```

Comment:
Since the SingleFileUpload component already does the error handling (shows an alert w/retry icon) this error dialog is no longer needed. The error was that the response was different so the parsing was failing and the state wasn't restored.